tests/test_path.py

```diff
@@ -1,4 +1,4 @@
-from os import scandir
+from os import name, scandir, symlink
```

Comment:
Detail (!) but feels a bit unnecessary to have a module-level variable named just `name` :) Maybe just `import os` and use `os.name` etc. for these, or `from os import name as os_name`, wdyt?

Reply:
done, same for path >>os_path